BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -29,9 +33,14 @@ public DiagnosticsTelemetryModule()
             // Adding a dummy queue sender to keep the data to be sent to the portal before the initialize method is called
             this.Senders.Add(new PortalDiagnosticsQueueSender());
 
+            this.FileDiagnosticsSender = new FileDiagnosticsSender();
+            this.Senders.Add(this.FileDiagnosticsSender);
```

Comment:
we are likely missing logs before initialization. Probably need similar approach as portalqueue to keep them in memory until a config is available.